xarray/tests/test_sparse.py

```diff
@@ -380,10 +390,18 @@ def test_dataarray_property(prop):
         (do("stack", z={"x", "y"}), True),
         (do("transpose"), True),
         # TODO
-        # isel_points
-        # sel_points
         # set_index
         # swap_dims
+        param(
+            do("broadcast_equals", make_xrvar({"x": 10, "y": 5})),
+            False,
+            marks=xfail(reason="https://github.com/pydata/sparse/issues/270"),
```

Comment:
This issue is fixed, so we should be able to re-enable these tests after the next sparse release.